client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -11,14 +11,15 @@ import Gridicon from 'gridicons';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
 import { compact, flatMap, get, isEmpty, zip } from 'lodash';
+import moment from 'moment';
```

Comment:
If we `localize( connect()( Component ) )` instead of `connect()( localize( Component ) )` then we get `moment` as a prop available in `connect()` as `ownProps.moment`. Then we can keep this import out

Reply:
Thanks for the tip! I've implemented it now.